nautobot/extras/templates/extras/graphqlquery.html

```diff
@@ -48,9 +48,23 @@
 {% block javascript %}
 {{ block.super }}
 <script>
+    function fillQueryOutput(data) {
+        const queryOutput = document.querySelector('#query_output');
+
+        [...queryOutput.childNodes].forEach(childNode => childNode.remove());
+
+        const code = document.createElement('code');
+        code.classList.add('language-json');
+        code.textContent = JSON.stringify(data, undefined, 2);
+
+        queryOutput.appendChild(code);
+
+        hljs.highlightElement(code);
+    }
+
     function test_query() {
```

Comment:
This is no longer working for me in firefox. When I click the `Execute` button I get this error in the console:

```
Uncaught TypeError: document.querySelector(...) is null
    test_query http://localhost:8080/extras/graphql-queries/46021bfc-3827-4c25-bc9f-2ecc6039d5f7/?tab=main:2985
    onclick http://localhost:8080/extras/graphql-queries/46021bfc-3827-4c25-bc9f-2ecc6039d5f7/?tab=main:1
```

Reply:
Looks like this is the issue maybe:

```
document.querySelector('#query_variables')
null 
```